views/components/Certification.js

```diff
@@ -1,72 +1,396 @@
-              combine your reading, listening, speaking and writing skills to
-              perform academic tasks.
-            </p>
-          </li>
-          <li className="process__step process__step_deploy">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deploy">
-            <h3>
-              <spana>3</spana>
-              &mdash;Second place
-            </h3>
-            <p>
-              Got second prize for the competitive algorithm contest of the
-              university
-            </p>
-          </li>
-          <li className="process__step process__step_deliver">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deliver">
-            <h3>
-              <spana>4</spana>
-              &mdash;First place
-            </h3>
-            <p>
-              Highschool, got First prize for the competitive algorithm contest
-              of the city.
-            </p>
-          </li>
-        </ul>
+import React, { useState, Fragment } from 'react'
+import styled from 'styled-components/macro'
+import Icon from '~/views/components/common/Icon'
+import { CERTIFICATIONS } from '~/consts/pages'
+import { INIT_CERT_ACTIVE_ITEM } from '~/consts/initState'
+import PropTypes from 'prop-types'
+
+const Certification = () => {
+  const [itemSelected, setSelectedItem] = useState(INIT_CERT_ACTIVE_ITEM[0])
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <header className="common-section__header">
+          <h2>Certifications and Prizes</h2>
+          <h4></h4>
+        </header>
+        <div className="grid-container">
+          <List className="certification">
+            <Research
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Develop
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deploy
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deliver
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+          </List>
+        </div>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
+
+const Research = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <Fragment>
+      <ResearchItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[0])}>
+        <ItemIcon
+          id={INIT_CERT_ACTIVE_ITEM[0]}
+          itemSelected={itemSelected}
+          name="c"
+        />
+      </ResearchItem>
+
+      <ResearchInfo itemSelected={itemSelected}>
+        <Header>
+          <span>1</span> &mdash; Bachelor of Science
```

Comment:
try to put text contents through parent component
using props like header = Bachelor of Science and info = Graduating from HCM University of Science, Advanced Program in Computer Science.

Reply:
Updated

![5](https://user-images.githubusercontent.com/7389488/70741429-d3b89c80-1d4d-11ea-913c-a6026849964b.PNG)
![6](https://user-images.githubusercontent.com/7389488/70741438-d74c2380-1d4d-11ea-98b5-940d071cc257.PNG)
![7](https://user-images.githubusercontent.com/7389488/70741441-d9ae7d80-1d4d-11ea-8e38-6c02e34e17f2.PNG)